src/main/resources/com/cloudbees/jenkins/GitHubPushTrigger/help.html

```diff
@@ -0,0 +1 @@
+This job will be triggered if jenkins will receive PUSH GitHub hook from repo defined in scm section
```

Comment:
this text will be injected to help-div as html, so no matter - it will be `<div><div>text..` or `<div>text...` i think. It looks the same for me in UI